objc/A0LockReactModule.m

```diff
@@ -97,10 +97,16 @@ @implementation A0LockReactModule
     });
 }
 
+RCT_EXPORT_METHOD(getDelegationToken:(NSDictionary *)options callback:(RCTResponseSenderBlock)callback) {
```

Comment:
I thought we can keep the api consistent with auth0.js and node-auth0. 

https://github.com/auth0/auth0.js/blob/3f5b58325d3ebad3a2ded4e851afedf3024a33ba/README.md#delegation-token-request

Should i change it?